### PR TITLE
Fix signup page

### LINKED ISSUE #1428: Unable to signup into calendso
### Issue Summary

Unable to signup into calendso, while running via docker

### Steps to Reproduce

1. Run docker-compose to launch calendso
2. http://localhost:3000/signup fails to render with the following error
3. 404 ERROR

This page does not exist.
The username localhost/signup is still available.

### Technical details

- docker-compose, calendso:latest

### PR DESCRIPTION
**Pull request by Jenietoc**
## What does this PR do?

In this PR we fixed the link that led to create an account and we also added a condition so that the signup page would load with the token set to null.

Fixes #1428

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [X] It was tested by creating accounts and logging in with them.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code and corrected any misspellings
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pages/auth/signup.tsx [hunk 1]
```diff
@@ -141,52 +141,57 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
...
+        trpcState: ssr.dehydrate(),
+      },
     };
-  }
-  const verificationRequest = await prisma.verificationRequest.findUnique({
-    where: {
-      token,
-    },
-  });
+  } else {
```
Comment (miguelnietoa) [inline pages/auth/signup.tsx]: This `else` statement can be removed because there is a `return` inside `if (!token)`.

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
Looks good to me, but I think the `else` statement is not needed. 👍🏻

**Comment by miguelnietoa** [comment]
Hey @acvalencia, could you give a code review of these changes, please?

Best!

File: pages/auth/signup.tsx (hunk 1 above)
Comment (Jenietoc) [reply pages/auth/signup.tsx]: perfect, I'll change it.

**Review by miguelnietoa — APPROVED** [review]

File: pages/auth/signup.tsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ export default function Signup({ email }: Props) {
                   className="block w-full px-3 py-2 mt-1 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-black focus:border-black sm:text-sm"
                 />
               </div>
-              <div className="flex rtl:space-x-reverse space-x-2">
+              <div className="flex space-x-2 rtl:space-x-reverse">
```
Comment (acvalencia) [inline pages/auth/signup.tsx:118]: What happens if this change is not added?

Comment (Jenietoc) [reply pages/auth/signup.tsx:118]: This change was made "automatically" by the library that was added to hierarchize the classes in Tailwind.  You can read a bit more https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

**Comment by PeerRich** [comment]
i don't think this is working because it lacks the entire signup logic that is currently in our closed source website

**Comment by Jenietoc** [comment]
> i don't think this is working because it lacks the entire signup logic that is currently in our closed source website

Considering the business logic, the only way to create an account is with an invitation token?

**Pull request closed without merging**